Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM openshift/jenkins-2-centos7
+COPY plugins.txt /opt/openshift/configuration/plugins.txt
+RUN /usr/local/bin/install-plugins.sh /opt/openshift/configuration/plugins.txt
```

Comment:
@AdamSaleh What is this for? If this is just splitting the slave config to a separate template do we need this?